### DISCUSSION TITLE
Request to reopen 0032 Python Code Samples

### DISCUSSION THREAD

**Discussion by ObiajuluM** [XRPLBounties/Proposals#69] [Polls]
We should reopen the [0032 Python Code Samples](https://github.com/XRPLBounties/Proposals/blob/main/closed_bounties/0032%20Python%20Code%20Samples.md) there are still some code samples that can be added, like NFTs support, Offer support, AMMs

**Comment by jonathanlei** [comment]
Could you specify what code samples (functions, scripts, etc) that you would like for the community?

**Reply by ObiajuluM** [reply]
Code samples for NFTs
Code samples for Offers

**Comment by pdp2121** [comment]
FYI, we are drafting a bounty proposal for code snippets, which would be example codes for new developers to interact with XRPL transaction types and objects, including NFTs and Offers. Once the PR for the proposal is created, please feel free to contribute your ideas.

Existing JavaScript code snippets:
https://github.com/XRPLF/xrpl.js/tree/main/packages/xrpl/snippets/src
Existing Python code snippets:
https://github.com/XRPLF/xrpl-py/tree/master/snippets

**Reply by ObiajuluM** [reply]
Nice
